app/main.py

```diff
@@ -1,13 +1,82 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error.get("line_number"),
+        "column": error.get("column_number"),
+        "message": error.get("text"),
+        "name": error.get("code"),
+        "source": "flake8",
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors":
+            [
+                {
+                    "line": error.get("line_number"),
+                    "column": error.get("column_number"),
+                    "message": error.get("text"),
+                    "name": error.get("code"),
+                    "source": "flake8",
+                }
+                for error in errors
+            ],
+        "path": file_path,
+        "status": "failed",
```

Comment:
status should be depend from errors, if errors is absent status should be success